phpunit.xml.dist

```diff
@@ -46,15 +46,15 @@
         </exclude>
     </groups>
 
-    <coverage>
+    <source ignoreSuppressionOfDeprecations="true">
         <include>
             <directory>./src/Symfony/</directory>
         </include>
         <exclude>
             <directory>./src/Symfony/Bridge/*/Tests</directory>
             <directory>./src/Symfony/Component/*/Tests</directory>
             <directory>./src/Symfony/Component/*/*/Tests</directory>
-            <directory>./src/Symfony/Contract/*/Tests</directory>
+            <directory>./src/Symfony/Contracts/*/Tests</directory>
```

Comment:
shouldn't this be fixed in older branches as well ?

Reply:
Probably yes, IIRC this doesn’t have any effect on older PHPUnit versions as we use them but it also doesn’t hurt to fix it.